README.md

```diff
@@ -96,3 +96,33 @@ First, make sure you have run the "Create some data" step above.
 1. Enter the web container: `docker-compose exec web bash`
 2. Build the indicies: `./manage.py esreindex` (You may need to pass the `--delete` flag)
 3. Precompile the nunjucks templates: `./manage.py nunjucks_precompile`
+
+## Development in vscode (beta)
+
+Install [vscode](https://code.visualstudio.com/) and the [Remote Development](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack) extension.
```

Comment:
Is it possible to add the extension in the vscode dockerfile directly?

Reply:
I don't think so, because this extension needs to run on the "host" vscode instance.